packages/bloc/test/bloc_test.dart

```diff
@@ -83,13 +87,21 @@ void main() {
               ),
             ),
           ).called(1);
-          expect(simpleBloc.currentState, 'data');
+          expect(simpleBloc.state, 'data');
         });
 
         simpleBloc.dispatch('event1');
         simpleBloc.dispatch('event2');
         simpleBloc.dispatch('event3');
       });
+
+      test('isBroadcast returns true', () {
+        expect(simpleBloc.isBroadcast, isTrue);
+      });
+
+      test('asBroadcastStream returns a broadcastStream', () {
```

Comment:
Could remove this test if ya remove `asBroadcastStream`, and ya can see here it's already a broadcast stream by default since it's using the subject under the hood which is itself a broadcast stream!